charts/replicated-library/README.md

```diff
@@ -166,9 +166,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### [Unreleased]
 
+### [0.9.0]
+
 ### Changed
 
-- altered helm-docs to generate documentation from values-example.yaml file.
+- Adding Global "Context" dictionaries for values and names with unique subkeys per object type to prevent collisions
+- Removing class directory and collapsing all templates into a single directory
```

Comment:
As a chart author do we expect people to have to change anything? Is there something we could add here specifically for chart authors?

While I know it's likely just David and Myself and I'm very familiar with this change it would be a good habit to get into so it's very easy for anyone using this to know if they need to make changes.

See the BREAKING: section of 0.7.0 release for an example of what I'm looking for.

Reply:
Is there actually anything breaking here for a consumer of the library chart? 🤔